tests/testthat/test-absolute_path_linter.R

```diff
@@ -200,3 +174,13 @@ test_that("returns the correct linting", {
     expect_lint(double_quote(path), NULL, linter)
   }
 })
+
+test_that("raw strings are handled correctly", {
+  skip_if_not_installed("base", "4.0.0")
```

Comment:
For consistency I think we should stick to a `getRVersion()` based skip.

Reply:
I wrote a helper for this instead... not a fan of the skip message `getRversion() < '4.0.0' is not TRUE`